plugins/techdocs/src/reader/components/Reader.tsx

```diff
@@ -45,10 +50,17 @@ type Props = {
   onReady?: () => void;
 };
 
+const useStyles = makeStyles<BackstageTheme>(() => ({
+  message: {
+    overflowWrap: 'anywhere',
```

Comment:
Heads up that [`overflow-wrap: anywhere` support isn't great right now](https://caniuse.com/mdn-css_properties_overflow-wrap_anywhere)